firebase-perf/src/main/java/com/google/firebase/perf/config/DeviceCacheManager.java

```diff
@@ -242,18 +250,22 @@ public Optional<Float> getFloat(String key) {
    *
    * @return whether provided value has been saved to device caching layer.
    */
-  public boolean setValue(String key, float value) {
+  public boolean setValue(String key, double value) {
```

Comment:
Now that we are using the same key, do we know the behavior of replacing a float value with a Double value in shared preferences? I don't think that should cause an issue, but still checking for completeness.

Reply:
Good point. I did check but I'm adding a test to make sure it's "documented".